src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -39,7 +39,7 @@
                                     {%- else -%}
                                             {{ redirect_route }}
                                     {%- endif %}
-                                    (<a href="{{ path('_profiler', { token: redirect.token }) }}">{{ redirect.token }}</a>)
+                                    (<a href="{{ path('_profiler', { token: redirect.token, panel: app.request.query.get('panel', 'request') }) }}">{{ redirect.token }}</a>)
```

Comment:
Not sure it should hardcode the default panel name (`request`). It's not needed right?